core/src/main/java/hudson/tasks/Fingerprinter.java

```diff
@@ -92,13 +93,40 @@
      */
     private final String targets;
 
+    /**
+     * Default null 'excludes' pattern as in Ant.
+     */
+    private String excludes = null;
+
+    /**
+     * Default ant exclusion
+     */
+    private Boolean defaultExcludes = true;
```

Comment:
```suggestion
    private boolean defaultExcludes = true;
```

Otherwise you'll have an NPE in `getDefaultExcludes()`